src/dags/importscripts/import_rdw.py

```diff
@@ -0,0 +1,31 @@
+import requests
+
+
+def download_data(endpoint: str, output_file: str) -> None:
```

Comment:
Er zit al een `download_file()` functie in de airflow repo (`dags/common/http.py`).